Library/Homebrew/software_spec.rb

```diff
@@ -306,9 +306,7 @@ def initialize(formula, spec)
 
     filename = Filename.create(formula, tag, spec.rebuild).bintray
 
-    # TODO: this will need adjusted when if we use GitHub Packages by default
-    path, resolved_basename = if (bottle_domain = Homebrew::EnvConfig.bottle_domain.presence) &&
-                                 bottle_domain.start_with?(GitHubPackages::URL_PREFIX)
+    path, resolved_basename = if spec.root_url.match?(GitHubPackages::URL_REGEX)
```

Comment:
Why `match?` instead of `start_with?` The latter is much faster.

Reply:
To permit `root_url "docker://ghcr.io/brewsci/bio"`